src/components/crossDevice/CrossDeviceLink/style.css

```diff
@@ -1,49 +1,146 @@
+.header {
+  margin-top: -71px;
+}
+
+.requestError {
+  margin-bottom: -11px;
+}
+
 .title{
   width: 312px;
   height: 80px;
   margin-left: 60px;
   margin-bottom: 16px;
 }
 
-.linkSection{
-  margin-top: 48px;
-  text-align: left;
+.smsSection {
+  height: 76px;
+  width: 432px;
+  margin-top: 56px;
+}
+
+.fieldLabel{
+  margin-bottom: 4px;
 }
 
-.linkTitle {
+.label {
   font-weight: 600;
-  margin-bottom: 5px;
+  text-align: left;
+  color: #2C3E4F;
+  float: left;
+}
+
+.sublabel {
+  color: #585E5F;
+  font-size: 14px;
+  margin-left: 4px;
+  font-weight: 400;
+  float: left;
+}
+
+.numberInputSection {
+	height: 48px;
+	width: 432px;
+  float: left;
+	border: 1px solid #667080;
+	border-radius: 4px;
+  border-right: none;
+	background-color: #FFFFFF;
+  margin-top: 4px;
+}
+
+.numberInputSection svg {
+   width: 33px;
+   height: 24px;
+}
+
+.numberError {
+  color: #DC2A2A;
+  font-size: 14px;
+  line-height: 20px;
+  float: left;
+  margin-top: 4px;
+}
+
+.sending:before {
+  z-index: 1;
+  position: absolute;
+  display: block;
+  top: 213px;
+  left: 366px;
+  bottom: 5px;
+  width: 16px;
+  height: 16px;
+  content: "";
+  opacity: 0.5;
+  background-repeat: no-repeat;
+  background-image: url('./assets/spinner.svg');
+
+  -webkit-animation-name: spin;
```

Comment:
Didn't you ever add in a precompiler to add vendor prefixes for you in the build? Stuff like this is horrible :'(

Reply:
yep, we have auto prefixer on the webpack config. Unless it stopped working on a webpack upgrade, which would be bad..